controllers/storagecluster/external_resources.go

```diff
@@ -375,7 +375,7 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				enableRookCSICephFS = true
 			} else if d.Name == cephRbdStorageClassName {
 				scc = newCephBlockPoolStorageClassConfiguration(instance)
-			} else if d.Name == cephRbdRadosNamespaceStorageClassName {
+			} else if strings.HasPrefix(d.Name, cephRbdRadosNamespaceStorageClassName) { // ceph-rbd-rados-namespace-<radosNamespaceName>
```

Comment:
please update the variable name to `cephRbdRadosNamespaceStorageClassNamePrefix` for better code readbility.

Reply:
Done